core/src/main/java/io/github/tcdl/api/Responder.java

```diff
@@ -58,7 +59,7 @@ public Message sendAck(Integer timeoutMs, Integer responsesRemaining) {
      * @return response message which was sent
      */
     public Message send(Payload responsePayload) {
-        AcknowledgeBuilder ackBuilder = this.messageFactory.createAckBuilder();
+        Acknowledge.AcknowledgeBuilder ackBuilder = this.messageFactory.createAckBuilder();
```

Comment:
Suppose that AcknowledgeBuilder is less code than Acknowledge.AcknowledgeBuilder.